### ISSUE TITLE
Separate config files for each tool

### ISSUE THREAD

**Issue by XVilka** [radare/radare2#9335]
To keep radare2rc to only radare2 tool, while support loading rabin2rc for rabin2, radiff2rc for radiff2, etc,

**Comment by radare** [comment]
R2 is the only one that run commands. What kind of rc scripts do u expect for the other tools? I cant imagine any... except env var or rarun profile..

**Comment by Maijin** [comment]
@radare Well radiff2 for example if someone fix this infamous issue about obeying colourscheme, we would need a way to provide colorscheme in a radiff2rc probably:  https://github.com/radare/radare2/issues/2141

**Comment by radare** [comment]
I would reuse r2rc for radiff instead of having a separste script

And i would prefer to implement per-file-format and such instead of adding more rcs. A way to run some commands depending on file type, etc like vim does

**Comment by XVilka** [comment]
Good idea. Not sure what is the good way to represent this in the current configuration file though.

**Comment by ret2libc** [comment]
This issue has been moved from radareorg/radare2 to radareorg/ideas as we are trying to clean our backlog and this issue has probably been created a long while ago. This is an effort to help contributors understand what are the actionable items they can work on, prioritize issues better and help users find active/duplicated issues more easily. If this is not an enhancement/improvement/general idea but a bug, feel free to ask for re-transfer to main repo. Thanks for your understanding and contribution with this issue.
